substrate/frame/revive/src/exec.rs

```diff
@@ -1524,8 +1522,17 @@ where
 		ContractInfoOf::<T>::contains_key(&address)
 	}
 
-	fn code_hash(&self, address: &H160) -> Option<H256> {
-		<ContractInfoOf<T>>::get(&address).map(|contract| contract.code_hash)
+	fn code_hash(&self, address: &H160) -> H256 {
+		<ContractInfoOf<T>>::get(&address)
+			.map(|contract| contract.code_hash)
+			.unwrap_or_else(|| {
+				if System::<T>::account_exists(&T::AddressMapper::to_account_id(address)) {
+					return H256::from(hex_literal::hex!(
+						"c5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470"
```

Comment:
I'd expect this to be a constant defined on the module level or somewhere, so that we can properly test against it?

Reply:
Makes sense